.swiftpm/xcode/xcshareddata/xcschemes/SimpleNetworkService.xcscheme

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should the scheme's name match the name of the target?

Reply:
I know that the target name and the schema name can be different - this does not affect the build process.
I agree that it's better to change the schema name to match the target, to provide a clearer and more unambiguous connection between the two.